### ISSUE TITLE
issue1-consistent-alt-attribute

### ISSUE THREAD

**Issue by EmmanuelMasango** [EmmanuelMasango/MyCV#1]
The alt attribute provides a text alternative for the image. This is important for accessibility, as it allows people who are unable to see the image to understand what it is.
